1.8 documentation updates: Concepts & menu structure (#11979)
cilium-team/cilium: Documentation/concepts/ipam/deep_dive.rst

.. only:: not (epub or latex or html)

    WARNING: You are looking at unreleased Cilium documentation.
    Please use the official rendered version released here:
    https://docs.cilium.io

###################
Technical Deep Dive
###################

Cilium Container Networking Control Flow
========================================

> The control flow picture below gives an overview about how the containers
> obtain its IP Address from the IPAM for different modes of Address Management
> that Cilium Supports.

Reviewer: ```suggestion
The control flow diagram below gives an overview on how containers
obtain their IP address from the IPAM for each different mode of Address Management
that Cilium Supports.
```
